server/data_structures/internal_datafacade.hpp

```diff
@@ -447,13 +448,15 @@ template <class EdgeDataT> class InternalDataFacade final : public BaseDataFacad
         }
 
         return m_static_rtree->IncrementalFindPhantomNodeForCoordinate(
-            input_coordinate, resulting_phantom_node_vector, number_of_results);
+            input_coordinate, resulting_phantom_node_vector, number_of_results, 1100, bearing, range);
```

Comment:
Meh, it's bad enough that we hard-code the max. distance, but having to hard-code it multiple times is just ugly.